buildpipeline/pipelines.json

```diff
@@ -70,6 +70,19 @@
             "PB_BuildType": null
           }
         },
+        {
+          "Name": "DotNet-CoreClr-Trusted-FreeBSD",
+          "Parameters": {
+            "Rid": "freebsd"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "FreeBSD",
+            "Type": "build/product/",
+            "SubType": "PortableBuild",
```

Comment:
I think this isn't actually a portable build but I wasn't sure what to put here.  This is just for telemetry.